content/blog/greenlake-platform-infrastructure-monitoring-with-apache-skywalking.md

```diff
@@ -0,0 +1,144 @@
+---
+title: GreenLake Platform Infrastructure monitoring with Apache SkyWalking
+date: 2022-12-13T08:52:15.545Z
+author: Thirukkannan M
+authorimage: /img/thirupp_192x192.jpg
+thumbnailimage: /img/thirupp_192x192.jpg
+disable: false
+tags:
+  - Infrastructure Monitoring
+  - Apache SkyWalking
+---
+**Overview**
+
+HPE GreenLake Central is an advanced software-as-a-service platform that provides you with a consistent cloud experience for all your applications and data - on-premises or off-premises. It provides you with insights and controls to manage your hybrid IT estate, complementing your use of public clouds and data centers. HPE GreenLake Central gives you the ability to choose where and how to place your workloads and data, and - through the services you purchase - enables you to monitor security, capacity, resource utilization, and costs.
+
+HPE GreenLake for Private Cloud Enterprise is a managed, pay-per-use, Private Cloud as a Service (PCaaS) solution that provides a self-service environment where you can implement a hybrid approach to run different kinds of bare metal, virtual machine, and container workloads. These workloads run from a single site without the need to create dedicated isolated environments and manage the underlying infrastructure, resulting in more flexibility and better ROI with the least operational overhead.
+
+SkyWalking is an open source observability platform used to collect, analyze, aggregate and visualize data from services and cloud native infrastructures. SkyWalking provides an easy way to maintain a clear view of your distributed systems, even across Clouds. It is a modern APM (Application performance Management), specially designed for cloud native, container based distributed systems.
+
+**Monitoring** is tooling or a technical solution that allows teams to watch and understand the state of their systems. Monitoring is based on gathering predefined sets of metrics or logs.
```

Comment:
These two points - Monitoring and Observability - kind of break up the flow of the blog post. I'd recommend that you put these in as Notes and not break up the information found in lines 14-19 and line 24. You can pull some of what you have in line 24 and add it into my suggested rewrite above.

Then, make a note below that says something like "Definitions:" and put Monitoring and Observability as bullets below that. By making them bullets, they will format in a much smaller font, thereby making it very clear that these are notes and not part of the main blog copy. Actually, very similar to what you did with Pre-requisite (which should be plural, by the way... "Pre-requisites". 

I'd also recommend putting in another **header** just before you get into the Definitions, Pre-requisites, and Steps to configure monitoring that says "**Let's get started**" using a Level 1 header.

So it would be something like this:

![image](https://user-images.githubusercontent.com/77496261/207458567-e2853157-717a-487b-acac-f1841f0b83fe.png)